.code-samples.meilisearch.yaml

```diff
@@ -83,10 +83,48 @@ get_update_1: |-
 get_all_updates_1: |-
   curl \
     -X GET 'http://localhost:7700/indexes/movies/updates'
-get_keys_1: |-
+get_one_key_1: |-
   curl \
-    -H "X-Meili-API-Key: 123" \
-    -X GET 'http://localhost:7700/keys'
+    -X GET 'http://localhost:7700/keys/d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4' \
+    -H 'Authorization: Bearer yourMasterKey'
+get_all_keys_1: |-
+  curl \
+    -X GET 'http://localhost:7700/keys' \
+    -H 'Authorization: Bearer yourMasterKey'
```

Comment:
Possibly nitpicking, but I find this authorization key a bit distracting. I assume this is a reference to something, what with the blank space between `Bearer` and `yourmasterkey`, but I'm not sure what.

Reply:
Ah, no, wait, I realised this is following the spec.

Perhaps we can add some `[]` to the master key value?

Reply:
```suggestion
    -H 'Authorization: Bearer [your_master_key]'
```